src/AppBar.js

```diff
@@ -89,17 +100,27 @@ AppBar.propTypes = {
   classes: PropTypes.object,
 
   /**
-   * Set as `true` if the AppBar should be fixed position.
+   * Affixes the AppBar to the top of the viewport. This prop is deprecated.
+   * Use `variant="fixed"` instead.
+   * @deprecated
    */
   fixed: PropTypes.bool,
 
   /**
    * String or Element to render within the offline warning container at the top of the app.
    */
   offlineWarning: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+
+  /**
+   * * relative - The AppBar stays at the top of the page.
```

Comment:
Did you mean to leave these extra asterisks in? I think they might break the documentation parsing.

Reply:
Yes, I did - I think that will create a markdown list, but I'm not sure.  Will check after release.

Reply:
Sweet